cmd/kubeadm/test/token_test.go

```diff
@@ -28,12 +28,11 @@ const (
 	TokenExpectedRegex = "^\\S{6}\\:\\S{16}\n$"
 )
 
-var kubeadmPath string
+var kubeadmPath = flag.String("kubeadm-path", filepath.Join(os.Getenv("KUBE_ROOT"), "cluster/kubeadm.sh"), "Location of kubeadm")
```

Comment:
They follow the test-cmd.sh pattern in most ways except for the fact that the tests themselves are written in Go instead of bash. As Derek pointed out, there was a lot of discussion around exactly where these tests should live and how they should work during the initial PR, and we came to this consensus then.

Also, in general the project's direction seems to favor replacing bash with Go code, so I'd prefer to keep these in Go.